app/src/lumen/java/com/cliqz/browser/vpn/VpnPanel.java

```diff
@@ -135,9 +140,22 @@ public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup c
         mVpnConnectButton.setOnClickListener(this);
         mVpnButtonTitle = mVpnConnectButton.findViewById(R.id.vpn_button_text_title);
         mVpnButtonDesc = mVpnConnectButton.findViewById(R.id.vpn_button_text_desc);
-        mVpnStatusText = view.findViewById(R.id.vpn_status_text);
         mWorldMap = view.findViewById(R.id.vpn_map);
         vpnTimer = view.findViewById(R.id.vpn_timer);
+        learnMoreLink = view.findViewById(R.id.learn_more_btn);
+        learnMoreLink.setOnClickListener(this);
+        featureOne = view.findViewById(R.id.vpn_feature_one);
+        featureTwo = view.findViewById(R.id.vpn_feature_two);
+        vpnCtaTitle = view.findViewById(R.id.vpn_cta_title);
+        if (VpnStatus.isVPNConnected()) {
+            featureOne.setCompoundDrawablesWithIntrinsicBounds(AppCompatResources.getDrawable(getContext(), R.drawable.ic_check), null, null, null);
+            featureTwo.setCompoundDrawablesWithIntrinsicBounds(AppCompatResources.getDrawable(getContext(), R.drawable.ic_check), null, null, null);
+        } else {
+            featureOne.setCompoundDrawablesWithIntrinsicBounds(AppCompatResources.getDrawable(getContext(), R.drawable.ic_cross), null, null, null);
+            featureTwo.setCompoundDrawablesWithIntrinsicBounds(AppCompatResources.getDrawable(getContext(), R.drawable.ic_cross), null, null, null);
+        }
```

Comment:
This happen so many times in our code that it will be probably better to create a class for this